openjdk.test.jck/src/native/makefile

```diff
@@ -75,94 +75,94 @@ COPYDIR=cp -r
 AND_IF_SUCCESSFUL=&&
 export AND_IF_SUCCESSFUL
 
-VPATH=$(SRCDIR)/src/share/lib/jni/:$(SRCDIR)/src/share/lib/atr/:$(SRCDIR)/src/share/lib/jvmti/:$(SRCDIR)/tests/api/javax_management/loading/data/archives/src/C/
+VPATH=$(SRCDIR)/src/share/lib/jni/:$(SRCDIR)/src/share/lib/atr/:$(SRCDIR)/src/share/lib/jvmti/:$(SRCDIR)/tests/api/javax_management/loading/data/archives/src/C/:$(SRCDIR)/src
 
 # Following are platform and compiler specific settings
 # Prior to jck10 the unix include files are in src/share/lib/jni/include/solaris
 # From jck10 the unix include files are in src/share/lib/jni/include/linux
 
 ifeq ($(PLATFORM),linux_x86-32)
 	CC=gcc
-	CFLAGS=-fPIC -I$(SRCDIR)/src/share/lib/jni/include/solaris -I$(SRCDIR)/src/share/lib/jni/include/linux
+	CFLAGS=-fPIC -I$(SRCDIR)/src/share/lib/jni/include/solaris -I$(SRCDIR)/src/share/lib/jni/include/linux -I$(SRCDIR)/src
```

Comment:
There are lots of code duplication here (not by this PR though).
It could be simplified via defining a variable `= -I$(SRCDIR)/src/share/lib/jni/include/solaris -I$(SRCDIR)/src/share/lib/jni/include/linux -I$(SRCDIR)/src` and using it in later `CFLAGS` defines.

Reply:
@JasonFengJ9 - thanks for pointing this out. I suggest let's make those changes to remove duplications in a separate PR (especially since this PR blocks the JCK PR for b09 which you've already approved.